File: src/leetcode/removeduplicatesfromsortedarraytwo/java/RemoveDuplicatesFromSortedArrayTwo.java
package leetcode.removeduplicatesfromsortedarraytwo.java;

import java.util.*;

public class RemoveDuplicatesFromSortedArrayTwo {
    public int removeDuplicates(int[] nums) {
        int i=0;
        for(int n:nums){
            if(i<2 || n!=nums[i-2]){
                nums[i++]=n;
            }
        }
        return i;
    }
}

File: src/leetcode/removeduplicatesfromsortedarraytwo/kotlin/RemoveDuplicatesFromSortedArrayTwo.kt
package leetcode.removeduplicatesfromsortedarraytwo.kotlin

class RemoveDuplicatesFromSortedArrayTwo {
    fun removeDuplicates(nums : IntArray) : Int {
        var i = 0;
        for (n in nums){
            if (i < 2 || n != nums[i - 2]) {
                nums[i++] = n
            }
        }
        return i
    }
}